include/libtorrent/alert_types.hpp

```diff
@@ -2496,7 +2537,7 @@ namespace libtorrent
 #undef TORRENT_DEFINE_ALERT_PRIO
 #undef TORRENT_CLONE
 
-	enum { num_alert_types = 89 };
+	enum { num_alert_types = 90 };
```

Comment:
Actually, I'm compiling with logging enabled. Before give up with this PR in the current form, I will explain a problem I'm facing (see in the PR's comment).